lib/src/text_view.dart

```diff
@@ -428,10 +435,7 @@ class _RichTextViewState extends State<RichTextView> {
             textAlign: widget.textAlign,
             textDirection: widget.textDirection,
             onTap: widget.onTap,
-            selectionControls: CustomTextSelectionControls(
-              originalText: widget.text,
-              visibleToOriginalIndexMap: visibleToOriginalIndexMap,
-            ),
+            contextMenuBuilder: contextMenuBuilder,
```

Comment:
Replaced deprecated selectionControls in favor of contextMenuBuilder to use adaptive controls